main.nf

```diff
@@ -597,7 +597,7 @@ process predict_peptides {
 
     script:
      """
-     python mhcflurry_predict_mztab.py ${allotypes} ${mztab_file} 'predicted_peptides.csv'
+     /opt/conda/envs/nf-core-mhcquant-1.1.0/bin/python mhcflurry_predict_mztab.py ${allotypes} ${mztab_file} 'predicted_peptides.csv'
```

Comment:
otherwise the host python version is picked, couldnt fix it so far

Reply:
also it doesnt find mhcflurry_predict_mztab.py in the mhcquant/bin folder

Reply:
nope its compatible with both python 2.7 and 3.4+

Reply:
the error I got was either it couldn't find the .py file in the /bin folder or it was picking this python: /opt/conda/bin/python

Reply:
Command executed:

  #!/usr/bin/env python
  mhcflurry_predict_mztab.py alleles.tsv all_features_merged_resolved.mzTab 'predicted_peptides.csv'

Command exit status:
  1

Command output:
  (empty)

Command error:
    File ".command.sh", line 2
      mhcflurry_predict_mztab.py alleles.tsv all_features_merged_resolved.mzTab 'predicted_peptides.csv'
                                       ^
  SyntaxError: invalid syntax

Reply:
ok so I tried chmod +x but it still doesn't seem to work. Have you checked this already?